Makefile

```diff
@@ -133,6 +134,19 @@ test-sim-backwards-compatible: ## Test uploading a file with lastest release (je
 
 ##@ Build
 
+.PHONY: storagenode-web
+storagenode-web:
```

Comment:
how about `storagenode-console` since we call it the `console` in other places?

Reply:
okay